CHANGELOG.next.asciidoc

```diff
@@ -114,6 +114,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add AWS s3_request metricset. {pull}10949[10949] {issue}10055[10055]
 - Add s3_daily_storage metricset. {pull}10940[10940] {issue}10055[10055]
 - Add `coredns` metricbeat module. {pull}10585[10585]
+- Allow module configurations to have variants or flavors {pull}9118[9118]
```

Comment:
Nitpick: this reads to me like `variants` and `flavors` are two different concepts. Since `variants` is used in the documentation you wrote, we could drop the word `flavors` here.

Reply:
@ycombinator Now there seem to be 2 lines :-)